FORMAT.md

```diff
@@ -30,10 +30,12 @@ The robot element is the root element of a robot model.
 - `version` (enum) Specifies the version number of the HRDF format used to define this file.  To allow parsing of v1.0.0 files, this defaults to `1.0.0` if not given.  The rules used to parse the file are defined by the given version.  For example, a `mass_offset` attribute on an actuator may cause a parsing error if this attribute is not given, or is set to `1.0.0`. Supported values are:
   - 1.0.0
   - 1.1.0
+  - 1.2.0
 
 **Optional Attributes**
 - `rot` (rotation matrix) specify the rotation of the base frame of the model; defaults to identity matrix.
 - `trans` (translation vector) specify the translation to the base frame of the model; defaults to (0,0,0)
+- `description` (string) human readable description of this robot
 
 **Example**
```

Comment:
All other places have "Example:" not "Example". We should add a colon for consistency.